### PR TITLE
Created NodeJS Cheatsheet

### PR DESCRIPTION
**Pull request by nikhilbuts**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [ ] Search for duplicates.
- [ ] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/nodeJs-cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+## Built In Methods
+
+| `fs` | read and write files on your file system |
+| `path` | combine paths regardless of which OS you're using |
+| `http` | make requests and create HTTP servers |
+| `https` | work with secure HTTP servers using SSL/TLS |
+| `events` | work with the EventEmitter |
+| `crypto` | cryptography tools like encryption and hashing |
+| `process` | information about the currently running process, e.g. process.argv for arguments passed in or process.env for environment variables |
```
Comment (crescentpartha) [inline CheatSheets/nodeJs-cheatsheet.md:17]: The table is broken.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Please, complete the full structure of this cheatsheet and follow [git-cheatsheet.md](https://github.com/crescentpartha/CheatSheets-for-Developers/blob/main/CheatSheets/git-cheatsheet.md) as a domo template of a cheatsheet. Thank you, @nikhilbuts

File: CheatSheets/nodeJs-cheatsheet.md (hunk 1 above)
Comment (nikhilbuts) [reply CheatSheets/nodeJs-cheatsheet.md:17]: @crescentpartha Can you please explain more about NodeJS Cheatsheet. So I can change based on that.

Thanks!

Comment (crescentpartha) [reply CheatSheets/nodeJs-cheatsheet.md:17]: - [node-cheatsheet](https://courses.cs.washington.edu/courses/cse154/19su/resources/assets/cheatsheets/node-cheatsheet.pdf) 
- [https://overapi.com/nodejs](https://overapi.com/nodejs) 
- [Node.js Documentation2](https://nodejs.org/api/documentation.html#json-output) 
- [Node.js Documentation1](https://nodejs.org/api/path.html#path_path_normalize_p)
You can check out those resources. then you create a structure in your own way.

**Review by crescentpartha — APPROVED** [review]
Approved.

**Pull request merged**